src/boilerplate/dispatcher.c

```diff
@@ -170,15 +170,12 @@ void apdu_dispatcher(command_descriptor_t const cmd_descriptors[],
         io_send_sw(SW_BAD_STATE);
     }
 
-    // We call the termination callback if given, but only if
-    // - the UX is "dirty", that is either
-    //     - there was some kind of UX flow with user interaction;
-    //     - background processing took long enough that the "Processing..." screen was shown.
-    // - and there has been an error and corresponding response has been sent
+    // We call the termination callback if given, but only if:
+    // - there was no UX flow with user interaction;
+    // - and background processing took long enough that the "Processing..." screen was shown.
     // Otherwise this is NBGL status callback that will be called
```

Comment:
I think it's worth making the comment more precise for future reference:

```suggestion
    // Otherwise, either nothing was shown on screen, or it is NBGL's responsibility
    // to clean up with the status callback
```

Reply:
Fixed